api/v1/serializers/pets-serializer.js

```diff
@@ -14,27 +14,24 @@ const petResourceKeys = _.keys(petResourceProp.attributes.properties);
 const petResourcePath = 'pets';
 const petResourceUrl = resourcePathLink(apiBaseUrl, petResourcePath);
 
-/**
- * The column name getting from database is usually UPPER_CASE.
- * This block of code is to make the camelCase keys defined in openapi.yaml be
- * UPPER_CASE so that the serializer can correctly match the corresponding columns
- * from the raw data rows.
+/*
+ * The column name getting from database is usually UPPER_CASE. This block of code is to make the
+ * camelCase keys defined in openapi.yaml be UPPER_CASE so that the serializer can correctly match
+ * the corresponding columns from the raw data rows.
  */
```

Comment:
Can you remove this comment or at least change this comment be more specific to this API? I realized we often use SQL alias to modify the retrieved data column name, so this logic is not really needed for the rest APIs.